### ISSUE TITLE
Request never returns - Broadcasting transaction to the blockchain...

### ISSUE THREAD

**Issue by leoplaw** [stoodkev/hive-keychain#87]
Using keychain to perform any action other than logging in, the request never returns

Broadcasting transaction to the blockchain...

returns no error, returns no success, permanently waiting for a response

Browsers: Brave, Firefox

**Comment by stoodkev** [comment]
Which dApp? Code? Node? 
I've just tested authentication, post, vote and memoDecode on PeakD, it s all working as intended.

**Comment by leoplaw** [comment]
peakd.com
code?
Node: default

I uninstalled and reinstalled on both Brave and Firefox.
Brave now works again.

Firefox, key chain donation screen.comes up with the donation page, buttons are half way off the bottom, cut off.
Clicking back to account set up, everything is overlayed on the donation screen. 

Screenshots attached.

![Screenshot_20210223_110442](https://user-images.githubusercontent.com/520581/108828580-4e727e00-75c7-11eb-9daf-194ff54390da.jpg)
![Screenshot_20210223_110548](https://user-images.githubusercontent.com/520581/108828585-503c4180-75c7-11eb-8854-a2c4cf434c80.jpg)

**Comment by stoodkev** [comment]
Hmm weird . Please give me your os and keychain version as well, i can't
reproduce the issue so far

**Comment by leoplaw** [comment]
Update: I uninstalled and reinstalled the keychain on Firefox and it worked again, flawlessly until today.
I was logged into PeakD, commenting and voting. I opened a PeakD link in a new tab, only for it to say I needed to log in, which I did. Voting and commenting was broken again. So, logout, and now cannot log in again as the plugin window never returns.
Screenshot attached.

![Screenshot_20210227_164631](https://user-images.githubusercontent.com/520581/109392438-53636480-791c-11eb-9f68-4320ac42d2c7.png)

Operating System: openSUSE Tumbleweed 20200819
KDE Plasma Version: 5.19.4
KDE Frameworks Version: 5.73.0
Qt Version: 5.15.0
Kernel Version: 5.8.0-1-default
OS Type: 64-bit
Processors: 8 × Intel® Core™ i7-2670QM CPU @ 2.20GHz
Memory: 15.6 GiB of RAM
Graphics Processor: NVC4

Firefox: 85.0.2 (64-bit)

Hive Keychain: 1.11.0.0
